src/geo/lng_lat_bounds.js

```diff
@@ -61,12 +61,12 @@ class LngLatBounds {
     }
 
     /**
-     * Extend the bounds to include a given LngLat or LngLatBounds.
+     * Extend the bounds to include a given LngLatLike or LngLatBoundsLike.
      *
-     * @param {LngLat|LngLatBounds} obj object to extend to
+     * @param {LngLatLike|LngLatBoundsLike} obj object to extend to
```

Comment:
Update the doc to specify the bounds can be extended to include a point or another bounds.

Reply:
This should also be `LngLat|LngLatLike|LngLatBounds|LngLatBoundsLike` right?

Reply:
not sure if it needs to be `LngLat|LngLatLike|LngLatBounds|LngLatBoundsLike` or `LngLatLike|LngLatBoundsLike`.
I have updated the input parameters to be`extend(obj: LngLatLike | LngLatBoundsLike)`but  I think each of the `LngLatLike` and `LngLatBoundsLike` would cover `LngLat` and `LngLatBounds` based on their definition:

`export type LngLatBoundsLike = LngLatBounds | [LngLatLike, LngLatLike] | [number, number, number, number];` & 
`export type LngLatLike = LngLat | {lng: number, lat: number} | {lon: number, lat: number} | [number, number];`